src/main/java/uk/gov/hmcts/reform/ccd/document/am/configuration/V1MediaType.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.ccd.document.am.configuration;
+
+import org.springframework.http.MediaType;
+
+public class V1MediaType extends MediaType {
```

Comment:
Vendor specific media types recommended to be introduced when second version is introduced.
Sorry, I noticed this only recently.